CONTRIBUTING.md

```diff
@@ -1,9 +1,7 @@
 # Contributing to BioJulia
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-This document contains information relevant for people who want to contribute to BioJulia repositories,
-which are hosted in the [BioJulia Organization](https://github.com/BioJulia) on
-GitHub.
+This document contains information relevant to people who want to contribute to BioJulia repositories, which are hosted in the [BioJulia Organization](https://github.com/BioJulia) on GitHub.
 
```

Comment:
Here, [sembr](https://sembr.org) actually does work, though. To me, having approximately a clause per-line is more readable in a diff.